### PR TITLE
test: interop jwt vc ecdsa

### PR DESCRIPTION
**Pull request by mishasizov-SK**
Part of #3289

### CODE REVIEW

**Comment by sudeshrshetty** [comment]
please squash your commits

**Comment by sudeshrshetty** [comment]
please follow these [guidelines](https://github.com/hyperledger/aries-framework-go/blob/main/.github/CONTRIBUTING.md#pull-request) for your pull request.

File: test/bdd/features/verifiable_credential.feature [hunk 1]
```diff
@@ -38,6 +38,15 @@ Feature: Issue Verifiable Credential
 
 @interop_jwt_verifiable
   Scenario: Load and verify Interop JWT credentials
-    When loading interop credential number "1" and verify it
-    And  loading interop credential number "2" and verify it
-    And  loading interop credential number "3" and verify it
+    When loading interop credential number "1" signed using "Ed25519" and verify it
```
Comment (sudeshrshetty) [inline test/bdd/features/verifiable_credential.feature:41]: what are the meaning of these numbers? can we have credential file name instead?

Comment (mishasizov-SK) [reply test/bdd/features/verifiable_credential.feature]: sure, added

**Review by Baha-sk — APPROVED** [review]
please sign the commit to pass all checks, but it LGTM

**Review by sudeshrshetty — APPROVED** [review]

**Review by Baha-sk — APPROVED** [review]
lgtm

**Comment by rolsonquadras** [comment]
Fixes https://github.com/hyperledger/aries-framework-go/issues/3287

**Pull request merged**